app/screens/thread/thread.js

```diff
@@ -126,7 +126,7 @@ class Thread extends PureComponent {
         } = this.props;
         const style = getStyle(theme);
         let content;
-        if (this.props.threadLoadingStatus.status === RequestStatus.STARTED) {
+        if (this.props.threadLoadingStatus.status === RequestStatus.STARTED && !this.hasRootPost()) {
```

Comment:
@enahum Rest of the posts are not loaded and are we sure we want to show it with just root post?

If we do have root post and another post from that thread we might see a flash of content appearing after loading rest of the posts.

Reply:
it still gains on perceived performance.. the way it was it feels way too slow

Reply:
Maybe we can have the `renderFooter` method render a spinner in that case? That way we can see the root post, but we can still see that there's content loading?

Reply:
Also I can't really think of a case where we have the root post and not the rest of the thread, maybe that'll be a case with the bidirectional scrolling

Reply:
Actually I do agree with @hmhealey on the footer, I'll work on that

Reply:
Ya that's true. Odds of having broken context << gain from showing the root post

Reply:
> Also I can't really think of a case where we have the root post and not the rest of the thread, maybe that'll be a case with the bidirectional scrolling

Posts in store because of permalink view/search results/pinned posts etc.

Reply:
> Actually I do agree with @hmhealey on the footer, I'll work on that

Sounds good to me 👍